frontend/components/modal/FakeUsers.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+const { fakeUsers: { fakeUsersVisible } } = useModal()
+const { t } = useI18n()
+const { getMaybeMe, refreshMaybeMe } = useSession()
+const { isAuthenticated, signIn, signOut } = await useMSAL()
+const localePath = useLocalePath()
+
+const prefix = 'ModalFakeUsers'
+const tt = (s: string) => t(`${prefix}.${s}`)
+
+// TODO(grady) move this to an env-specific location, omit the component if empty.
+const hardcodedLocalUsers = [{
+  id: 'user.c9a959a1c4d6b8be1fc1',
+  email: 'rmi-localtest-a@siliconally.org',
+  password: 'CQdCZd9lq2HOnFB',
+}, {
+  id: 'user.4aed45b19c7ef24e105c',
+  email: 'rmi-localtest-b@siliconally.org',
+  password: 'Dq7CLBX90nFtNk8',
+}]
```

Comment:
Do you see any risk with these being public? I think it's probably fine, but you could also `sops` encode credentials and then decrypt them at start-up time. Also, do we need all this fake user infra? What's wrong with just logging out and logging in as other users? It just seems like more pages and more code (and now, more translation) than the limited utility it provides.

Reply:
^^

Reply:
Yeah I agree I don't see any risk here.

I'm confident you will find this valuable - I already have.